frontend/src/pages/auth/styles.module.scss

```diff
@@ -71,26 +73,23 @@
     justify-content: center;
   }
 
-  .auth .logo {
-    z-index: 2;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 149px;
-    height: 56px;
-    margin: 0.1px 130px 32px 0.1px;
-    font-size: 21px;
-    background: var(--blue-300);
-    border-radius: 14px;
+  .auth-name {
+    width: 280px;
+    text-align: start;
+  }
+
+  .auth-name .secondary {
+    display: none;
+  }
```

Comment:
Remove duplicate style selectors. `.auth-name` and `.auth-name .secondary` are used multiple times